resource-group.tf

```
@@ -2,7 +2,7 @@
 resource "azurerm_resource_group" "rg" {
   name     = "${var.product}-${var.env}"
   location = var.location
-  tags     = merge(local.common_tags, map("lastUpdated", timestamp()))
+  tags     = merge(var.common_tags, map("lastUpdated", timestamp()))
```

this was for a removed feature, and will cause a change on every apply which is not recommended
```suggestion
  tags     = var.common_tags
```